bpf/lib/nodeport.h

```diff
@@ -692,14 +692,22 @@ int tail_nodeport_nat_ingress_ipv6(struct __ctx_buff *ctx)
 
 	ret = snat_v6_rev_nat(ctx, &target);
 	if (IS_ERR(ret)) {
-		/* In case of no mapping, recircle back to main path. SNAT is very
-		 * expensive in terms of instructions (since we don't have BPF to
-		 * BPF calls as we use tail calls) and complexity, hence this is
-		 * done inside a tail call here.
-		 */
-		ctx_skip_nodeport_set(ctx);
-		ep_tail_call(ctx, CILIUM_CALL_IPV6_FROM_NETDEV);
-		ret = DROP_MISSED_TAIL_CALL;
+		if (ret == NAT_PUNT_TO_STACK ||
+		    /* DROP_NAT_NO_MAPPING and DROP_NAT_UNSUPP_PROTO
+		     * are both unwanted behavior from a revNAT
+		     * context. Let's continue to passing it up to the
```

Comment:
Use either "in a context" or "from a perspective"
```suggestion
		     * are both unwanted behavior in a revNAT
		     * context. Let's continue to passing it up to the
```